_includes/analytics.html

```diff
@@ -1,19 +1,18 @@
-	<!-- Digital Analytics Program roll-up, see https://analytics.usa.gov for data -->
-	<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA?subagency=TTS"></script>
-
 	<!-- Google Analytics -->
 	<script>
-	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-	  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-	  ga('create', '{{ site.google_analytics_ua }}', 'gsa.gov');
-
-	  // anonymize user IPs (chops off the last IP triplet)
-	  ga('set', 'anonymizeIp', true);
+		window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+		ga('create', '{{ site.google_analytics_ua }}', 'gsa.gov');
+		ga('send', 'pageview');
+		// anonymize user IPs (chops off the last IP triplet)
+		ga('set', 'anonymizeIp', true);
 
-	  // forces SSL even if the page were somehow loaded over http://
-	  ga('set', 'forceSSL', true);
+		// forces SSL even if the page were somehow loaded over http://
+		ga('set', 'forceSSL', true);
 
-	  ga('send', 'pageview');
+		ga('send', 'pageview');
 	</script>
+	<script async src='https://www.google-analytics.com/analytics.js'></script>
+
+<!-- Digital Analytics Program roll-up, see https://analytics.usa.gov for data -->
+	<script async type="text/javascript" id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA?subagency=TTS"></script>
```

Comment:
Change to the second question mark to an ampersand: `https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&subagency=TTS`

Reply:
Fixed in https://github.com/uswds/uswds-site/commit/8dee9ef4396ad7ad894ea35a23d2e1b2e019c9d4